contents/handbook/engineering/ee-setup.md

```diff
@@ -63,6 +63,23 @@ This setup is best if you want to iterate quickly on frontend changes. You get t
   - `export KAFKA_HOSTS=localhost:9092`
 - Run PostHog: `./bin/start`
 
+### Running ClickHouse, Kafka and Zookeeper on a Cloud Server
+
+This is useful if you have a Apple Silicon Mac.
+
+1. Get SSH access to any cloud server, e.g. a droplet from Digital Ocean.
+2. `ssh -L 8123:localhost:8123 -L 9000:localhost:9000 -L 9440:localhost:9440 -L 9009:localhost:9009 -L 9092:localhost:9092 root@DROPLET.IP.ADDRESS`
```

Comment:
nit: if you use an env for the droplet IP address it's easier to copy pasta this whole line or make it an alias

Reply:
If you know what you're doing, it's really easy to make it into an alias no matter what :). 

I think this is fine for now to be honest. If I'd add an ENV it'd also have `DUMP.YOUR.IP.HERE`, but now you'd have to copy paste two things. Not sure if a great improvement in the end.